geometry_msgs/Point[] area_points
geometry_msgs/Point[] nav_points
---
bool area_empty
---
